core/src/main/java/com/nisovin/magicspells/spells/targeted/SpawnEntitySpell.java

```diff
@@ -174,6 +176,7 @@ public SpawnEntitySpell(MagicConfig config, String spellName) {
 		useCasterName = getConfigBoolean("use-caster-name", false);
 		addLookAtPlayerAI = getConfigBoolean("add-look-at-player-ai", false);
 		allowSpawnInMidair = getConfigBoolean("allow-spawn-in-midair", false);
+		attackSpellCancelsDefaultAttack = getConfigDataBoolean("attack-spell-cancels-default-attack", true);
```

Comment:
Name is a bit long, and doesn't really fit other option names. It's also a bit misleading, since it cancels all damage, not the "default" attack. Perhaps `cancel-attack`? The option description can handle mentioning it only applies if there's an `attack-spell`. Also, none of the other boolean options on this spell support replacement, so it's a bit out of place that this one does, but that's fine if you want to keep it that way.